ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_paddle_activation.py

```diff
@@ -389,7 +389,25 @@ def test_paddle_celu(
         available_dtypes=helpers.get_dtypes("float"),
     ),
 )
-def test_paddle_rrelu(*, dtype_and_x, on_device, fn_tree, frontend, test_flags):
+def test_paddle_rrelu(
+    *, 
+    dtype_and_x, 
+    on_device, 
+    fn_tree, 
+    frontend, 
+    test_flags,
```

Comment:
Thanks! Looks good. But I see `lint` failures due to this part. Please use `pre-commit` to format your code. For more info: https://unify.ai/docs/ivy/overview/contributing/setting_up.html#pre-commit

Reply:
Done